test/ui-test/src/screens/StatusWalletScreen.py

```diff
@@ -121,3 +131,45 @@ def generate_new_account(self, account_name: str, password: str):
     def verify_account_name_is_present(self, account_name: str):
         verify_text_matching(MainWalletScreen.ACCOUNT_NAME.value, account_name)
         
+    def send_transaction(self, account_name, amount, token, chain_name, password):
+        click_obj_by_name(MainWalletScreen.SEND_BUTTON_FOOTER.value)
+        
+        accounts = get_obj(SendPopup.HEADER_ACCOUNTS_LIST.value)
```

Comment:
Maybe we can create a new `find_account` method and reuse the loop in next steps.

Reply:
I did a click_repeater function